app/build.gradle

```diff
@@ -34,12 +34,14 @@ android {
     defaultConfig {
         minSdkVersion 16
         targetSdkVersion 25
-        versionCode 36
+        versionCode 38
```

Comment:
I thought this change was caused by the android studio update.
if you think this version up is unnecessary, I will be revert this.